src/main/java/com/study/realworld/domain/user/ui/UserRestController.java

```diff
@@ -0,0 +1,31 @@
+package com.study.realworld.domain.user.ui;
+
+import com.study.realworld.domain.user.application.UserJoiningService;
+import com.study.realworld.domain.user.dto.UserJoinRequest;
+import com.study.realworld.domain.user.dto.UserJoinResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import javax.validation.Valid;
+
+@Controller
+public class UserRestController {
+
+    private static final Logger log = LoggerFactory.getLogger(UserRestController.class);
+
+    private final UserJoiningService userJoiningService;
+
+    public UserRestController(final UserJoiningService userJoiningService) {
+        this.userJoiningService = userJoiningService;
+    }
+
+    @PostMapping("/api/users")
+    public ResponseEntity<UserJoinResponse> join(@Valid @RequestBody final UserJoinRequest userJoinRequest) {
+        log.info(userJoinRequest.toString());
```

Comment:
아마 들어오는 request의 기록용일듯한데, **굳이 여기에 로깅을 한다면** 이게 정확히 뭐가 들어온 것인지 표시 해주는 것이 좋을 것 같아요.
```suggestion
        log.info("join request: {}", userJoinRequest.toString());
```
이런 느낌으로...?

Reply:
로깅도 처음 적용해봤는데 찬스님 방법이 좋을 것 같네요 👍